app/frontend/actions.js

```diff
@@ -141,6 +141,7 @@ module.exports = ({setState, getState}) => {
       mnemonic: CARDANOLITE_CONFIG.CARDANOLITE_DEMO_WALLET_MNEMONIC,
       mnemonicValidationError: undefined,
       walletLoadingError: undefined,
+      focus: 'goBtn',
```

Comment:
the name 'focus' created the impression that the element is focused while this variable is set to 'goBtn' but actually goBtn remains focused even when focus is undefined, after calling clearFocus in components.js. Therefore I would choose a less generic name for this property, perhaps "focusTo"?